app/backend/app.py

```diff
@@ -256,39 +253,6 @@ def config():
     )
 
 
-async def _build_usecase_clients(
```

Comment:
This function has been added to be able to work with multiple indexes based on different usecases that are approached using different paths. Removing this will break this functionality.